docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/create-space-syntax.md

```diff
@@ -1,7 +1,7 @@
 # CREATE SPACE 语法
 
 ```ngql
-CREATE SPACE space_name
+CREATE SPACE (IF NOT EXIST) space_name
     (space_option,...)
```

Comment:
<space_option>

Reply:
`space_option` is optional so we use `[]`here. I changed the above session as follow:
```
CREATE SPACE [IF NOT EXIST] <space_name>
    [(partition_num = $your_num, replica_factor = $your_num)]
```

Reply:
> `space_option` is optional so we use `[]`here. I changed the above session as follow:
> 
> ```
> CREATE SPACE [IF NOT EXIST] <space_name>
>     [(partition_num = $your_num, replica_factor = $your_num)]
> ```

not correct.

CREATE SPACE [IF NOT EXIST] <space_name>
    [(partition_num = $part_num , replica_factor = $raft_copy)]